### DISCUSSION TITLE
Visual settings in the admin interface

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#533] [General]
Hi,
I have a custom 'Textarea' type 'Memberfield' form field that I can display in the 'EME/Members' admin interface. When I edit a member, this field appears at the end in a frame that can be adjusted in height but not in width. I looked in the inspector and couldn't find any CSS to modify it. Only the 'nodynamicupdates' class appears.
How can I make this area editable in width and not just in height?

**Comment by f14m07** [comment]
I have found the answer:
- edit the custom field
- Fill HTML field attributes with 'style="resize: both"'

**Reply by liedekef** [reply]
Indeed, that's why I created the html-part in custom fields :-)
